### PR TITLE
feat(geo): setup

### PR DESCRIPTION
**Pull request by keiya01**
- Setup back-end architecture in Nest.js
  - Added template implementation for Tile API
  - Added template implementation for GraphQL
- Setup ci-bff workflow
- Setup Dockerfile to deploy to GCR and run Cloud run.

### CODE REVIEW

**Review by pyshx — APPROVED** [review]
Thanks, LGTM!

File: bff/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+  },
+};
```
Comment (rot1024) [inline bff/.eslintrc.js:25]: Use [eslint-config-reearth](https://github.com/reearth/eslint-config-reearth)

**Review by rot1024 — COMMENTED** [review]
There are too many unique features that do not involve a backend to be called "bff". Since this server includes many map-related functions such as tile distribution and geocoding, why not name it "geo"?

**Comment by keiya01** [comment]
> There are too many unique features that do not involve a backend to be called "bff". Since this server includes many map-related functions such as tile distribution and geocoding, why not name it "geo"?

Renamed.
[chore(geo): rename bff to geo](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/3/commits/213f10d066a582da29a7c1ba43dcc12e9d302570)

File: bff/.eslintrc.js (hunk 1 above)
Comment (keiya01) [reply bff/.eslintrc.js]: Fixed.
[chore(geo): use eslint-config-reearth](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/3/commits/4ffa1736afa89a166ad7741c547b8fb77df0ab7a)

**Review by rot1024 — APPROVED** [review]

**Pull request merged**